tests/classes/event_properties/biosero/test_automation_system_name.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from lighthouse.classes.event_properties.definitions import RunID
+from lighthouse.classes.event_properties.definitions.biosero import AutomationSystemName, RunInfo
+from lighthouse.constants.fields import (
+    FIELD_CHERRYTRACK_LIQUID_HANDLER_SERIAL_NUMBER,
+    FIELD_EVENT_RUN_ID,
+    FIELD_CHERRYTRACK_USER_ID,
+    FIELD_CHERRYTRACK_AUTOMATION_SYSTEM_MANUFACTURER,
+    FIELD_CHERRYTRACK_AUTOMATION_SYSTEM_NAME,
+)
+from http import HTTPStatus
+
+
+def test_robot_serial_number_valid(app):
+    assert AutomationSystemName(RunInfo(RunID({FIELD_EVENT_RUN_ID: 1}))).valid() is True
```

Comment:
I forget the reasoning so happy to zoom if it would be quicker/easier but if this is the alias it does not read as clearly as `is_valid()`.

Reply:
yes, renamed to is_valid()

Reply:
Done